content/blog/2022/08/camel-karavan-3.18.1/index.md

```diff
@@ -0,0 +1,77 @@
+---
+title: "Karavan Preview Release 3.18"
+date: 2022-08-19
+authors: [mgubaidullin]
+categories: ["Releases","Tooling"]
+preview: "Karavan 3.18 - cloud-native integration toolkit"
+---
+
+![logo](toolkit.png)
+
+The next significant milestone is Karavan development. Karavan still comes with two main options: VS Code extenson and cloud-native integration toolkit for Kubernetes/OpenShift (FKA cloud/serverless mode). 
+
+Disclamer:
+
+Karavan version is aligned with Camel version.
+Karavan 3.18 NOT LTS and still an experimental preview release.
+Versions alignment is required to avoid DSL compatibility matrix between Camel and Karavan.
+
+# Cloud-native Itegration Toolkit
+
+Starting from this preview release Karavan could be deployed to OpenShift (Kubernetes version is comming) to design, build, deploy and monitor integrations.
+
+## Architecture
+Karavan cloud-native integration toolkit is a user application to manage projects, design and monitor integrations, Tekton pipeline to build and deploy integrations.
+
+![architecture](./karavan-cloud-native.png)
+
+## Projects
+Karavan manages projects stored in Git repository
+
+![theme](./karavan-projects.png)
+
+![theme](./github-projects.png)
+
+## Build and deploy
+Karavan uses Tekton Pipelines to build and deploy integrations. Pipeline generates canonical camel-quarkus maven project, then maven package command makes compilation, image build and deployment. No magic!
```

Comment:
The pipeline is of course fully customisable so users can start using Karavan in their own existing CI/CD platforms.